web_search_with_and/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2015 Andrius Preimantas <andrius@versada.lt>
```

Comment:
Can you change the copyright character to 'Copyright' to comply the template? https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py